app/jobs/short_form_attachment_job.rb

```diff
@@ -35,4 +30,14 @@ def perform(application_id, file_id)
       error: error_message,
     )
   end
+
+  # now that file is saved in SF, remove file binary and mark as delivered
+  def self.update_file_on_success(file, application_id)
```

Comment:
actually this should already be logic that we have somewhere, right?

Reply:
discussed over zoom, this is just moved from line 19. The only new addition is we're also clearing out the error value.